docs/source/user-guide/getting-started.rst

```diff
@@ -0,0 +1,28 @@
+***************
+Getting started
+***************
+
+Before starting the tutorials, consider reviewing
+:doc:`how to install and update conda build <../../install-conda-build>`
+and :doc:`conda build concepts <../../concepts/index>`.
+
+As you explore, you may find our :doc:`resources <../../resources/index>`
+collection helpful.
+
+.. _prereqs:
+
+Prerequisites
+=============
+
+Before starting the tutorials, we recommend that
+you install `Miniconda or Anaconda <https://docs.anaconda.com/anaconda/install/>`_, conda build, and git.
+
+Conda build and git can each be installed using conda.
```

Comment:
Recommend pre-pending with "After you've installed Miniconda or Anaconda, you can use conda to install conda build and git."